--- dawn/contact.html.tmp	2016-03-17 16:37:11.000000000 +0000
+++ dViewer/contact.html.tmp	2016-03-17 10:18:01.000000000 +0000
@@ -16,7 +16,7 @@
 
 		<!-- Text Logo -->
 		<h1 id="logo" class="grid_4">
-			<a href="http://org.eclipse.ui.intro/runAction?pluginId=org.dawnsci.intro&amp;class=org.dawnsci.intro.actions.OpenWebBrowserAction&amp;url=http://www.dawnsci.org"><img src="images/dawn-logo.png" width="300" alt="http://www.dawnsci.org"></img></a>
+			<a href="http://org.eclipse.ui.intro/runAction?pluginId=org.dawnsci.intro&amp;class=org.dawnsci.intro.actions.OpenWebBrowserAction&amp;url=http://www.dawnsci.org"><img src="images/dViewer-logo-408x176.png" width="300" alt="http://www.dawnsci.org"></img></a>
 		</h1>		
 		<!-- Navigation Menu -->
 		<ul id="navigation" class="grid_8">
