--- dawn/inprogress.html.tmp	2016-03-17 16:37:11.000000000 +0000
+++ dViewer/inprogress.html.tmp	2016-03-17 10:18:01.000000000 +0000
@@ -13,7 +13,7 @@
 <div id="wrapper" class="container_12 clearfix">
 		<!-- Text Logo -->
 		<h1 id="logo" class="grid_4">
-			<a href="http://www.dawnsci.org"><img src="images/dawn-logo.png" width="300"></img></a>
+			<a href="http://www.dawnsci.org"><img src="images/dViewer-logo-408x176.png" width="300"></img></a>
 		</h1>
 		
 		<!-- Navigation Menu -->
