--- dawn/perspectives.html.tmp	2016-03-17 16:37:11.000000000 +0000
+++ dViewer/perspectives.html.tmp	2016-03-17 10:18:01.000000000 +0000
@@ -11,7 +11,7 @@
 <div id="wrapper" class="container_12 clearfix">
 
 	<h1 id="logo" class="grid_4">
-			<a href="http://org.eclipse.ui.intro/runAction?pluginId=org.dawnsci.intro&amp;class=org.dawnsci.intro.actions.OpenWebBrowserAction&amp;url=http://www.dawnsci.org"><img src="images/dawn-logo.png" width="300" alt="http://www.dawnsci.org"></img></a>
+			<a href="http://org.eclipse.ui.intro/runAction?pluginId=org.dawnsci.intro&amp;class=org.dawnsci.intro.actions.OpenWebBrowserAction&amp;url=http://www.dawnsci.org"><img src="images/dViewer-logo-408x176.png" width="300" alt="http://www.dawnsci.org"></img></a>
 	</h1>
 	
 	<!-- Navigation Menu -->
