--- dawn/plugin.xml.tmp	2016-03-21 10:42:32.165073320 +0000
+++ dViewer/plugin.xml.tmp	2016-03-21 10:44:23.157542443 +0000
@@ -7,34 +7,42 @@
          point="org.eclipse.core.runtime.products">
       <product
             application="org.eclipse.ui.ide.workbench"
-            name="DAWN Science">
+            name="dViewer">
          <property
                name="appName"
-               value="DAWN Science">
+               value="dViewer">
+         </property>
+         <property
+               name="startupForegroundColor"
+               value="FFFFFF">
+         </property>
+         <property
+               name="startupMessageRect"
+               value="10,10,200,20">
          </property>
          <property
                name="startupProgressRect"
-               value="5,240,590,15">
+               value="0,513,1224,15">
          </property>
          <property
                name="windowImages"
-               value="icons/Temp-16x16-icon.png,icons/Temp-48x48-icon.png">
+               value="icons/dViewer-16x16-icon.png,icons/dViewer-48x48-icon.png">
          </property>
          <property
                name="aboutImage"
-               value="icons/Dawn-210x238-about.png">
+               value="icons/dViewer-210x238-about.png">
          </property>
          <property
                name="introBrandingImage"
-               value="product:icons/Temp-3Col-Splash.png">
+               value="product:icons/dViewer-3Col-Splash.png">
          </property>
          <property
                name="introTitle"
-               value="Welcome to DAWN">
+               value="Welcome to dViewer">
          </property>
          <property
                name="aboutText"
-               value="Data Analysis WorkbeNch &#x0A;Copyright, 2010 - _DATEYEAR_, Diamond Light Source&#x0A;&#x0A;Version _VERSION_">
+               value="dViewer (Diffraction Viewer based on Data Analysis Workbench [DAWN])&#x0A;Copyright, 2010 - _DATEYEAR_, DAWN Collaboration&#x0A;&#x0A;Version _VERSION_">
          </property>
          <property
                name="preferenceCustomization"
