--- dawn/feature.xml.tmp	2016-03-17 16:37:11.836347449 +0000
+++ dViewer/feature.xml.tmp	2016-03-17 10:18:01.000000000 +0000
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="org.dawnsci.base.product.feature"
-      label="DAWN Science"
+      label="dViewer"
       version="_VERSION_.qualifier"
       provider-name="DAWN Collaboration">
 
-   <description url="http://www.example.com/description">
-      [Enter Feature Description here.]
+   <description url="http://www.dawnsci.org/">
+      Diffraction viewer
    </description>
 
-   <copyright url="http://www.example.com/copyright">
-      Copyright (c) _DATEYEAR_ Diamond Light Source Ltd.
+   <copyright url="http://www.dawnsci.org/">
+      Copyright (c) _DATEYEAR_ DAWN Collaboration
    </copyright>
 
    <license url="http://www.eclipse.org/legal/epl-v10.html">
 </feature>
